zulipterminal/config/keys.py

```diff
@@ -460,6 +460,10 @@ class KeyBinding(TypedDict):
         f"Stream information (press {KEY_BINDINGS['STREAM_INFO']['keys'][0]}"
         f" to view info of a stream)"
     ),
+    "msg_info": (
+        f"Message information (press {KEY_BINDINGS['MSG_INFO']['keys'][0]}"
+        f" to view info of a message)"
+    ),
```

Comment:
Ah, I meant to point out here that the f-string should not be necessary for the second string :)